plugins/janus_streaming.c

```diff
@@ -3162,6 +3182,10 @@ static json_t *janus_streaming_process_synchronous_request(janus_streaming_sessi
 			json_t *iface = json_object_get(root, "rtspiface");
 			json_t *threads = json_object_get(root, "threads");
 			json_t *failerr = json_object_get(root, "rtsp_failcheck");
+			json_t *reconnect_delay = json_object_get(root, "reconnect_delay");
+			json_t *session_timeout = json_object_get(root, "session_timeout");
+			json_t *rtsp_timeout = json_object_get(root, "rtsp_timeout");
+			json_t *rtsp_conn_timeout = json_object_get(root, "rtsp_conn_timeout");
```

Comment:
The new properties should be defined in `rtsp_parameters`, which allows us to do automatic type and validity checks. There's no validation done here, meaning I could send `rtsp_timeout` as a string and you're doing a `json_integer_value` later.

Reply:
Thanks, I didn't see that struct and everything worked well without this validation too.